test/model/content_test.dart

```diff
@@ -141,10 +141,20 @@ void main() {
     // TODO test group mentions and wildcard mentions
   });
 
-  testParseInline('parse Unicode emoji',
+  testParseInline('parse Unicode emoji, encoded in span element',
     // ":thumbs_up:"
     '<p><span aria-label="thumbs up" class="emoji emoji-1f44d" role="img" title="thumbs up">:thumbs_up:</span></p>',
-    const UnicodeEmojiNode(text: ':thumbs_up:'));
+    const UnicodeEmojiNode(emojiName: ':thumbs_up:', codepoints: '\u{1f44d}'));
+
+  testParseInline('parse Unicode emoji, encoded in span element, multiple codepoints',
+    // ":transgender_flag:"
+    '<p><span aria-label="transgender flag" class="emoji emoji-1f3f3-fe0f-200d-26a7-fe0f" role="img" title="transgender flag">:transgender_flag:</span></p>',
+    const UnicodeEmojiNode(emojiName: ':transgender_flag:', codepoints: '\u{1f3f3}\u{fe0f}\u{200d}\u{26a7}\u{fe0f}'));
+
+  testParseInline('parse Unicode emoji, not encoded in span element',
+    // "🪿"
+    '<p>🪿</p>',
+    const TextNode('\u{1fabf}'));
```

Comment:
nit: clearest to read if these either all use the `\u` notation, or all have the literal emoji in the source

(either way it's the same string, just two different ways in the source code of expressing it)